compiler/rustc_middle/src/mir/syntax.rs

```diff
@@ -668,8 +668,10 @@ pub enum TerminatorKind<'tcx> {
     ///
     /// [#71117]: https://github.com/rust-lang/rust/issues/71117
     Call {
-        /// The function that’s being called.
-        func: Operand<'tcx>,
+        /// The function that’s being called, including the `Span` of the
+        /// function, without the dot and receiver
+        /// e.g. `foo(a, b)` in `x.foo(a, b)`
```

Comment:
Now that we have separate spans for the arguments, wouldn't it make a lot more sense to make this just the `foo`? That's what I would expect from a `Spanned<Operand>` -- that the span only covers that operand.

Reply:
Hmm, good point. Since that is not as straightforward, I'll leave that for the future and go ahead and close this.

Thanks for taking a look!